sdk/mixedreality/azure-mixedreality-authentication/src/main/java/com/azure/mixedreality/authentication/MixedRealityStsClientBuilder.java

```diff
@@ -0,0 +1,402 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.mixedreality.authentication;
+
+import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpHeader;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.policy.AddHeadersPolicy;
+import com.azure.core.http.policy.BearerTokenAuthenticationPolicy;
+import com.azure.core.http.policy.CookiePolicy;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpLoggingPolicy;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.http.policy.UserAgentPolicy;
+import com.azure.core.util.ClientOptions;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.mixedreality.authentication.implementation.MixedRealityStsRestClientImpl;
+import com.azure.mixedreality.authentication.implementation.MixedRealityStsRestClientImplBuilder;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * This class provides a fluent builder API to help aid the configuration and instantiation of {@link
+ * MixedRealityStsClient MixedRealityStsClients} and {@link MixedRealityStsAsyncClient MixedRealityStsAsyncClient}, call {@link
+ * #buildClient() buildClient} and {@link #buildAsyncClient() buildAsyncClient} respectively to construct an instance of
+ * the desired client.
+ *
+ * @see MixedRealityStsAsyncClient
+ * @see MixedRealityStsClient
+ */
+@ServiceClientBuilder(serviceClients = {MixedRealityStsClient.class, MixedRealityStsAsyncClient.class})
+public final class MixedRealityStsClientBuilder {
+    private static final String APP_CONFIG_PROPERTIES = "azure-mixedreality-authentication.properties";
```

Comment:
Update the name of the constant - `MIXED_REALITY_STS_PROPERTIES`.

Reply:
Doh. Good catch. Thanks!